purr/templates/404-error.html

```diff
@@ -0,0 +1,27 @@
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","contentSize":"","wideSize":"1200px"}} -->
+<div class="wp-block-group" style="padding-top:0;padding-bottom:0"><!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"25%","style":{"spacing":{"padding":{"right":"var:preset|spacing|30","left":"var:preset|spacing|30"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-column" style="padding-right:var(--wp--preset--spacing--30);padding-left:var(--wp--preset--spacing--30);flex-basis:25%"><!-- wp:group {"style":{"position":{"type":"sticky","top":"0px"},"layout":{"selfStretch":"fixed","flexSize":"20%"}},"className":"left-sidebar-sticky","layout":{"type":"default"}} -->
+<div class="wp-block-group left-sidebar-sticky"><!-- wp:template-part {"slug":"left-sidebar","theme":"purr"} /--></div>
+<!-- /wp:group --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"50%","style":{"spacing":{"padding":{"right":"0","left":"0"}},"border":{"left":{"color":"#f2f2f2","width":"1px"},"top":[],"right":{"color":"#f2f2f2","width":"1px"},"bottom":[]}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-column" style="border-right-color:#f2f2f2;border-right-width:1px;border-left-color:#f2f2f2;border-left-width:1px;padding-right:0;padding-left:0;flex-basis:50%"><!-- wp:heading {"textAlign":"center","level":1,"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|purr-blue"}}},"typography":{"fontStyle":"normal","fontWeight":"600","fontSize":"50px"},"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|30"}}},"textColor":"purr-blue","fontFamily":"system-font"} -->
+<h1 class="wp-block-heading alignfull has-text-align-center has-purr-blue-color has-text-color has-link-color has-system-font-font-family" style="padding-top:var(--wp--preset--spacing--80);padding-bottom:var(--wp--preset--spacing--30);font-size:50px;font-style:normal;font-weight:600"><strong>404 </strong>Error</h1>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"align":"center"} -->
+<p class="has-text-align-center">Go to <a href="https://irreplaceable-ape-592207.instawp.xyz/" data-type="link" data-id="https://irreplaceable-ape-592207.instawp.xyz/">Homepage</a></p>
```

Comment:
Let's use a similar wording to other themes instead of adding a link here. [Reference](https://github.com/WordPress/community-themes/blob/49b35be328591af07aa851bf67b17b54da032654/blue-note/patterns/hidden-404.php#L17)